### PR TITLE
KMK find initial

### PR DESCRIPTION
**Pull request by kmkinnaird**

### CODE REVIEW

File: repytah/utilities.py [hunk 1]
```diff
@@ -212,83 +206,63 @@ def find_initial_repeats(thresh_mat, bandwidth_vec, thresh_bw):
...
+                    end_i_over = start_i_over + (bw - 1)
+                    end_j_over= start_j_over + (bw - 1)
 
-                    i_eshin = np.vstack((end_i_shin + ones_no - K,
-                                         end_i_shin)).T
-                    j_eshin = np.vstack((end_i_shin + ones_no,
-                                         end_j_shin)).T
-                    eint_lst = np.column_stack((i_eshin, j_eshin, K.T))
+                    ematch_i_over = end_i_over - K + 1 
+                    ematch_j_over = end_i_over - 1
```
Comment (aliu-12) [inline repytah/utilities.py:236]: why is it `ematch_j_over = end_i_over - 1` and not `ematch_j_over = end_i_over + 1`

File: repytah/utilities.py [hunk 2]
```diff
@@ -212,83 +206,63 @@ def find_initial_repeats(thresh_mat, bandwidth_vec, thresh_bw):
                     # NOTE: the middle interval may NOT exist
                     # Vector of 1's that is the length of the number of
                     # overlapping intervals. This is used a lot.
-                    ones_no = np.ones(num_overlaps)
+                    ones_no = np.ones(num_overlaps, int)
```
Comment (aliu-12) [inline repytah/utilities.py:209]: I'm a little bit confused why sometimes we use `ones_no` and sometimes we can just use scalar 1? 
Ex. Line 235 uses 1 in `ematch_i_over = end_i_over - K + 1` but Line 257 uses `ones_no` in `end_i_over - start_j_over - K + ones_no`. Both are using it on the same variable.

File: repytah/utilities.py (hunk 1 above)
Comment (kmkinnaird) [reply repytah/utilities.py:236]: This was an error on my part

File: repytah/utilities.py [hunk 3]
```diff
@@ -212,83 +206,63 @@ def find_initial_repeats(thresh_mat, bandwidth_vec, thresh_bw):
...
                     # Add the new left overlapping intervals to the full list
                     # of left overlapping intervals
                     sint_all = np.vstack((sint_all, sint_lst))
 
                     # 2b) Right Overlap
 
-                    end_i_shin = start_i_shin + (bw - 1)
-                    end_j_shin = start_j_shin + (bw - 1)
+                    end_i_over = start_i_over + (bw - 1)
+                    end_j_over= start_j_over + (bw - 1)
```
[the remark this answers is not in the document]
Comment (kmkinnaird) [reply repytah/utilities.py:233]: fixed

File: repytah/utilities.py (hunk 2 above)
Comment (kmkinnaird) [reply repytah/utilities.py:209]: Good point. I'll remove the ones_no

**Pull request merged**